tilequeue/command.py

```diff
@@ -978,18 +978,23 @@ def tilequeue_prune_tiles_of_interest(cfg, peripherals):
     with psycopg2.connect(redshift_uri) as conn:
         with conn.cursor() as cur:
             cur.execute("""
-                select x, y, z
+                select x, y, z, tilesize
                 from tile_traffic_v4
                 where (date >= dateadd(day, -{days}, current_date))
-                  and (z between 10 and 16)
+                  and (z between 0 and 15)
                   and (x between 0 and pow(2,z)-1)
                   and (y between 0 and pow(2,z)-1)
                 group by z, x, y
                 order by z, x, y;""".format(days=redshift_days_to_query))
             n_trr = cur.rowcount
-            for (x, y, z) in cur:
+            for (x, y, z, tile_size) in cur:
```

Comment:
Indeed! I do like parens - mainly because I can't remember precedence rules or, as in this case, like being reminded that we're destructuring a tuple.